cmd/chisel/cmd_cut.go

```diff
@@ -80,13 +93,21 @@ func (cmd *cmdCut) Execute(args []string) error {
 		return err
 	}
 
+	arch := cmd.Arch
+	if arch == "" {
+		arch := goToDebArch[runtime.GOARCH]
+		if arch == "" {
+			return fmt.Errorf("cannot infer architecture from current platform %s, see the --arch option", runtime.GOARCH)
+		}
+	}
+
 	archives := make(map[string]archive.Archive)
 	for archiveName, archiveInfo := range release.Archives {
 		openArchive, err := archive.Open(&archive.Options{
 			Label:    archiveName,
 			Version:  archiveInfo.Version,
 			CacheDir: cache.DefaultDir("chisel"),
-			Arch:     "amd64", // TODO Option for this, implied from running system
+			Arch:     arch,
```

Comment:
Then, let's allow the `Arch` to be empty for `archive.Open` and do the inferring there (also with testing), instead of having the detection logic at the CLI layer.